frontend/src/store/creator/reducer.js

```diff
@@ -101,7 +104,26 @@ export function reducer(state, action) {
 		case SELECT_EDGE:
 			return {
 				...state,
-				selectedEdge: action.edge,
+				selectedEdges: action && action.edge !== null ? [action.edge] : [],
+			}
+		case TOGGLE_EDGE_SELECTION:
+			if (!action || action.edge === null) {
```

Comment:
Could be `== null`, then we guard against null and undefined.

Reply:
But checking action is not required, since you already did `switch(action.type)`